test/acceptance/behave/components/ngsiv2/entities/update_existing_entity_attributes/update_existing_entity_attributes_part3.feature

```diff
@@ -479,7 +478,6 @@ Feature: update an attribute by entity ID if it exists using NGSI v2. "PATCH" -
     And properties to entities
       | parameter        | value       |
       | attributes_name  | temperature |
-      | attributes_value | 45          |
```

Comment:
This means that the attribute has no value and not ype? I mean, something like this at API level:

PUT /v2/entities/room/attrs/temperature

{
  "metadata": {
    "my_meta": { "value": 5687, "type": "nothing" }
  } 
}